src/topology/uniform_space/pi.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2019 Patrick Massot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Patrick Massot
+
+Index products of uniform spaces
```

Comment:
```suggestion
Indexed product of uniform spaces
```

Reply:
@PatrickMassot This suggestion is still open.